code/game/g_weapon.c

```diff
@@ -104,55 +105,61 @@ static void WP_FireHyperspanner(gentity_t *ent, qboolean alt_fire) {
     gentity_t*	nearest = NULL;
     int			count = 0;
     float		nearestd = 65000;
-    vec3_t		dVec, end;
-    vec3_t		mins = { -40, -40, 0 }, maxs = { 40, 40, 0 };
     struct list	classnames;
 
+    int radius = 96;
```

Comment:
Consider making this a #define similar to HYPERSPANNER_RATE, to keep parameterisation consistent?